aces/pipeline_scripts/override_tclean_commands.json

```
@@ -2307,11 +2307,18 @@
         "width": "",
         "threshold": "0.0153Jy",
         "cyclefactor": 3.0
+      },
+      "spw35": {
+        "nchan": -1,
+        "width": "0.4883123MHz",
```

it might be better to leave this as 
```suggestion
        "width": "",
```